tasks/steps/blocks/set_control_instance.yml

```diff
@@ -5,7 +5,7 @@
   when: not not_expelled_instance
```

Comment:
What about to return import_tasks instead include_tasks for set_not_expelled_instance?